mappings/net/minecraft/world/PersistentStateManager.mapping

```diff
@@ -6,12 +6,22 @@ CLASS net/minecraft/class_26 net/minecraft/world/PersistentStateManager
 	METHOD <init> (Ljava/io/File;Lcom/mojang/datafixers/DataFixer;)V
 		ARG 1 directory
 		ARG 2 dataFixer
-	METHOD method_120 get (Ljava/util/function/Supplier;Ljava/lang/String;)Lnet/minecraft/class_18;
+	METHOD method_120 load (Ljava/util/function/Supplier;Ljava/lang/String;)Lnet/minecraft/class_18;
 		ARG 1 factory
-		ARG 2 id
+		ARG 2 fileName
```

Comment:
Does this include the file extension? If not, then I'd leave it "id", or "name".